src/providers/execution/contracts/cs_fee_distributor.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from web3.types import BlockIdentifier
+
+from ..base_interface import ContractInterface
+
+logger = logging.getLogger(__name__)
+
+
+class CSFeeDistributor(ContractInterface):
+    abi_path = "./assets/CSFeeDistributor.json"
+
+    def pending_to_distribute(self, block: BlockIdentifier = "latest") -> int:
```

Comment:
Rename to kwarg to `block_identifier` in all contract calls. That's required for proper working cache.